assets/less/bootstrap/button-groups.less

```diff
@@ -85,7 +84,7 @@
 .btn-group > .btn:focus,
 .btn-group > .btn:active,
 .btn-group > .btn.active {
-  z-index: 2;
+  z-index: 1;
```

Comment:
I think it'd be better to set the `z-index` at https://github.com/apache/couchdb-fauxton/blob/master/assets/less/formstyles.less#L350 instead.
This way it's obvious what we are overriding whenever we upgrade bootstrap.

Reply:
@Antonio-Maranhao sounds good. I will do this tonight!

Reply:
@Antonio-Maranhao required a `!important` since bootstrap took hire precedence with more focused scope. Based on specificity: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity ... its okay practice in this case to use it since Bootstrap is a "3rd party" stylesheet.